doc/python/imshow.md

```diff
@@ -20,7 +20,7 @@ jupyter:
     name: python
     nbconvert_exporter: python
     pygments_lexer: ipython3
-    version: 3.7.7
+    version: 3.7.11
```

Comment:
are there any changes in this file's output?

Reply:
No. Hadn't seen it update. I've reverted that now.